xrpl/models/requests/request.py

```diff
@@ -74,6 +74,10 @@ class RequestMethod(str, Enum):
     PING = "ping"
     RANDOM = "random"
 
+    # generic unknown/unsupported request
+    # (there is no XRPL analog, this model is specific to xrpl-py)
+    CUSTOM_REQUEST = "custom_request"
```

Comment:
wondering if we should define this const at all, since using it would always be an error. perhaps the new request should just have an empty, optional field there

Reply:
That'll mess up the rest of our typing - the `method` param requires a `TransactionType` enum type.

Reply:
in that case, let's prepend the value with `zz`, to further show it's nto a real request and to not have any collision with any future actual request

Reply:
I think the phrase "request" in there does that. But I can do that.